packages/db/src/loaders/resources/compilations/index.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  CompilationData,
+  LoadedSources,
+  IdObject,
+  WorkspaceRequest,
+  WorkspaceResponse
+} from "@truffle/db/loaders/types";
+
+import { AddCompilations } from "./add.graphql";
+export { AddCompilations };
```

Comment:
Why do we export this here as well as the places we imported it from in this file?

Reply:
Oh I see, this is the main index file.